src/Component/CategoryDetail/Component/ShowAll/ShowAll.js

```diff
@@ -0,0 +1,173 @@
+import React, { Component } from "react";
+import "./ShowAll.scss";
+// import Unisex from
+
+class ShowAll extends Component {
+  constructor() {
+    super();
+    this.state = {
+      showAllList: [],
+      currentCategory: 0,
+      showDropdown: false,
+      isArrowDown: true,
+      sortMethod: "최신순",
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://10.58.6.191:8000/products?sort=0&second=1", {
+      method: "GET",
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        this.setState({
+          showAllList: res.category_list,
+          src: res.category_list.main_image_url,
+        });
+      });
+  }
+
+  sortProduct = (e) => {
+    const sort = e.target.dataset.idx;
+    console.log(e.target.id);
+    fetch(`http://10.58.6.191:8000/products?sort=${sort}&second=1`)
+      .then((res) => res.json())
+      .then((res) => {
+        this.setState({
+          showAllList: res.category_list,
+          showDropdown: !this.state.showDropdown,
+          sortMethod: e.target.id,
+        });
+      });
+  };
+
+  handleMenuTab = (e) => {
+    // console.log(e);
+  };
+
+  filterDropdown = (e) => {
+    e.preventDefault();
+
+    this.setState({
+      showDropdown: !this.state.showDropdown,
+      isArrowDown: !this.state.isArrowDown,
+    });
+  };
+
+  handleFilter = (e) => {
+    // console.log(e);
+  };
+
+  render() {
+    const { showAllList, showDropdown, sortMethod } = this.state;
+    const { filterDropdown, sortProduct } = this;
+    // console.log(sortMethod);
+
+    return (
+      <div className="ShowAll">
+        <div className="header">
+          <div className="title">
+            <span>전체보기</span>
+          </div>
+          <div className="filter">
+            <button onClick={filterDropdown}>
+              <span>{sortMethod}</span>
+              <span>
+                {!showDropdown ? (
+                  <img
+                    src="images/icon/filterArrow.png"
+                    className="descBtn"
+                    alt="상품정렬버튼"
+                  />
+                ) : (
+                  <img
+                    src="images/icon/filterArrow.png"
+                    className="ascBtn"
+                    alt="상품정렬버튼"
+                  />
+                )}
```

Comment:
아래와 같이 변경해주면 가독성에 더 좋을 것 같습니다. 
```suggestion
                  <img
                    src="images/icon/filterArrow.png"
                    className={!showDropdown ? "descBtn" : "ascBtn" }
                    alt="상품정렬버튼"
                  />
```

Reply:
이런거 너무 좋아요...............
제 무지함을 이렇게 깨닿게 해주시다니.......!